tests/testprogs/stack_args.c

```diff
@@ -0,0 +1,8 @@
+// Declare enough args that some are placed on the stack
+void too_many_args(int a, int b, int c, int d, int e, int f, int g, int h) {}
```

Comment:
This adds a lot of warning to the build output as all args are unused, something like this will fix it:
```
#define ARG(x) int (x) __attribute((unused))

void too_many_args(ARG( a), ARG( b), ARG( c), ARG( d), ARG( e), ARG( f), ARG( g), ARG( h)) {}

int main(void)
```

Reply:
Oof, yes it does. Nice catch.